src/test/integration/nodes.e2e-spec.ts

```diff
@@ -276,7 +316,88 @@ describe('Node Service', () => {
   describe('Get Node Count', () => {
     it('should return node count', async () => {
       const count = await nodeService.getNodeCount(new NodeFilter());
-      expect(typeof count).toBe('number');
+      expect(typeof count).toStrictEqual('number');
```

Comment:
I think it would be great to test that count is a real value, not only that it is of type number.
Maybe this type of scenarios can be performed with a large mocked list of nodes